examples/CmdServer.hs

```diff
@@ -22,8 +22,8 @@ import qualified Data.Map.Strict as Map
 import qualified Streamly.Data.Array as Array
 import qualified Streamly.Data.Fold as Fold
 import qualified Streamly.Data.Parser as Parser
-import qualified Streamly.Data.Stream as Stream
-import qualified Streamly.Data.Stream.Concurrent as Concur
+import qualified Streamly.Internal.Data.Stream as Stream
+import qualified Streamly.Internal.Data.Stream.Concurrent as Concur
```

Comment:
Avoid internal imports, in case you have to import something internal then import it explicitly.